src/app/profile/profile.component.ts

```diff
@@ -1,16 +1,30 @@
 import { Component, OnInit } from '@angular/core';
+import { DataService } from '../shared/services/data.service';
+import { User } from '../shared/models/user.model';
```

Comment:
It doesn't look like this import is being used in the component yet, should we remove it?